doc/backend_vars.asciidoc

```diff
@@ -43,6 +43,16 @@ UPLOAD_INACTIVITY_TIMEOUT;integer;300;Specifies the inactivity timeout in second
 
 |====================
 
+.ZVM backend
+[grid="rows",format="csv"]
+[options="header",cols="^m,^m,^m,v",separator=";"]
+|====================
+Variable;Values allowed;Default value;Explanation
+ZVM_HOST;string;;Sets the remote host name.
+ZVM_GUEST;string;;Sets the remote user name.
```

Comment:
```suggestion
ZVM_GUEST;string;;Sets the remote username.
```
(Just for consistency with your other PR.)

Reply:
fixed in https://github.com/os-autoinst/os-autoinst/pull/1912